sw/device/tests/autogen/plic_all_irqs_test.c

```diff
@@ -460,6 +460,7 @@ bool test_main(void) {
   irq_global_ctrl(true);
   irq_external_ctrl(true);
   peripherals_init();
+  CHECK_DIF_OK(dif_rv_plic_reset(&plic));
```

Comment:
I wondered about that. I added it in because after removing the reset function call from the `dif_rv_plic_init()` I didn't want to change the functionality of any tests, but I will remove it now that you confirmed it is unnecessary.

Reply:
Fixed.